network/dag/bbolt_tree.go

```diff
@@ -60,6 +61,7 @@ func newBBoltTreeStore(db *bbolt.DB, bucketName string, tree tree.Tree) *bboltTr
 		bucketName:             bucketName,
 		tree:                   tree,
 		activeRollbackRoutines: new(uint32),
```

Comment:
is this one still used?

Reply:
https://github.com/nuts-foundation/nuts-node/blob/55fe9d4576f0d80f615a7a894b5d70c81b98992c/network/dag/bbolt_tree_test.go#L169-L171

it is still used here. But this is currently always zero and cannot be replaced by `store.numRollbacks`

Reply:
I intended to remove `activeRollbackRoutines`. Will fix.

Reply:
I thought it wasn't required any more, but it still was. I put it back.